docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     ports:
       - 4444:4444
     volumes:
-      - $HOME/.msf4:/root/.msf4
+      - /tmp/.msf4:/home/msf/.msf4
```

Comment:
Seems like the environment issue on my end. Tried this Dockerfile on different host and it works. My bad.